lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -70,15 +70,16 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     }
 
     // Overwrite the displayValue with opportunity's wastedMs
-    const displayEl = this.dom.find('.lh-audit__display-text', element);
+    const displayEl =
+      this.dom.find('span.lh-audit__display-text, div.lh-audit__display-text', element);
```

Comment:
this one's kind of unfortunate :/

I wonder why we have [`<span>`](https://github.com/GoogleChrome/lighthouse/blob/73af3c897db52f0ad80180fc97eaab0a36d26259/lighthouse-core/report/html/templates.html#L99) in one place and a [`<div>`](https://github.com/GoogleChrome/lighthouse/blob/73af3c897db52f0ad80180fc97eaab0a36d26259/lighthouse-core/report/html/templates.html#L136) in the other?

Reply:
> this one's kind of unfortunate :/
> 
> I wonder why we have [`<span>`](https://github.com/GoogleChrome/lighthouse/blob/73af3c897db52f0ad80180fc97eaab0a36d26259/lighthouse-core/report/html/templates.html#L99) in one place and a [`<div>`](https://github.com/GoogleChrome/lighthouse/blob/73af3c897db52f0ad80180fc97eaab0a36d26259/lighthouse-core/report/html/templates.html#L136) in the other?

actually, they always end up `display: flex` from [`.lh-audit__header`](https://github.com/GoogleChrome/lighthouse/blob/73af3c897db52f0ad80180fc97eaab0a36d26259/lighthouse-core/report/html/report-styles.css#L529-L530), so they could both be `<span>` or both be `<div>` and not have to have this double selector. The minor improvement is that the next person looking at this line doesn't have to wonder if it's significant that they're different types of elements, but up to you if that's worth anything :)

Reply:
Why use the tag name at all?
```suggestion
      this.dom.find('.lh-audit__display-text', element);
```

Reply:
> The minor improvement is that the next person looking at this line doesn't have to wonder if it's significant that they're different types of elements, but up to you if that's worth anything :)

`div` changed the layout.

![image](https://user-images.githubusercontent.com/4071474/105532152-11595a00-5cb0-11eb-99f9-554da8ba1dd3.png)

I guess you mistook `.lh-audit__header` to be the parent of this element, but it's the grandparent.

> Why use the tag name at all?

Because `title` doesn't exist on `Element`.

(also, this file is JS, not TS).

Reply:
> >  they always end up `display: flex` from `.lh-audit__header`, so they could both be <span> or both be <div> and not have to have this double selector.
> 
> `div` changed the layout.
> 
> ![image](https://user-images.githubusercontent.com/4071474/105532152-11595a00-5cb0-11eb-99f9-554da8ba1dd3.png)
> 
> I guess you mistook `.lh-audit__header` to be the parent of this element, but it's the grandparent.

Ah, whoops, sorry. Well, the `<div>` _can_ be a `<span>` because that's the one of those two cases I actually tested to see if it would work :)

Reply:
I wasn't able to confidently test it (quickly..) so I just tossed a TODO there. Also leaving the "remove .title" work to a different PR, so figured keeping this all to just type changes / should-be-noop runtime changes is ideal.